clients/src/test/java/org/apache/kafka/clients/admin/internals/AbortTransactionHandlerTest.java

```diff
@@ -52,6 +53,10 @@ public class AbortTransactionHandlerTest {
     @Test
     public void testInvalidBuildRequestCall() {
         AbortTransactionHandler handler = new AbortTransactionHandler(abortSpec, logContext);
+
+        assertEquals(handler.apiName(), "abortTransaction");
+        assertNotNull(handler.lookupStrategy());
```

Comment:
I'm not sure these checks add a lot of value. Especially in this test that is about building invalid calls.

Reply:
Removed